### ISSUE TITLE
Update Per-Bar Configuration UI

### ISSUE THREAD

**Issue by Tuller** [tullamods/Dominos#175]
The right click menus in Dominos work decently, but have the following flaws:

* They aren't super discoverable
* They're tiny

For the first issue, I should be able to add buttons to the drag frame to get users to show the options menu.

For the latter, I should make larger menus.  These menus should probably pop up in the middle of the screen, but shouldn't be part of the main Addons configuration interface because I want the frame to be movable.

**Comment by Goranaws** [comment]
You could add a tooltip to the drag frames that said "Right Click for config" (or something like that)

**Comment by Tuller** [comment]
They have those already

**Comment by Goranaws** [comment]
oh.. I'm a bit behind the times.. just getting back into the game.

**Comment by Tuller** [comment]
Right click menus are now a bit bigger, and feature a more minimal look than before.  I condensed the number of tabs on action bars down to three: Layout, Paging, and Advanced.  Paging now contains all of the bar offset options that were previously on a bunch of separate tabs, hopefully making it less necessary to swap back and forth.  I've also converted all paging sliders to drop downs, so that ordering is more consistent.  

Since paging sliders were the only non numeric sliders in use in Dominos, I was also able to add in edit boxes to each slider so that users can directly input values (which is much more intuitive than me telling people to mousewheel).

Showstate editing isn't implemented yet.  It will go on the advanced menu for the time being.  Bonus points if I also add in a custom paging option there, too.
